### PR TITLE
Bug 1326595, added Managing Users section

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1326595

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @bfallonf PTAL thanks!

File: admin_guide/manage_users.adoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+NAME                  IDP NAME        IDP USER NAME   USER NAME   USER UID
+htpasswd_auth:ashley  htpasswd_auth   ashley          ashley      75e4b80c-dbf1-11e5-8dc6-0e81e52cc949
+----
+====
+
+Note the UID between the two matches. If you attempt to change the
+authentication provider after starting to use OpenShift, the user names that
+overlap will not work because of the entries in the identity list, which will
+still point to the old authentication method.
+
```
Comment (bfallonf) [inline admin_guide/manage_users.adoc]: @ahardin-rh I got a little lost and had to reread the first sentence. Can I suggest:

"Note the matching UID between the two results."

I think the problem was that 'matches' was too far from what it's modifying (UID).

Also, from what I understand, are you talking about two different improper ways to delete a user? It seems it goes from deleting the data in the wrong order, but then talking about changing identity providers. I don't think i can see a link between the two... Maybe a little elaboration is needed.

**Comment by bfallonf** [comment]
@ahardin-rh Just the one comment from me (which I think is actually two comments...). Otherwise, LGTM :+1:

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../admin_guide/manage_users.html[Managing Users]
|Added the link:../admin_guide/manage_users.html[Managing Users] section.
%

File: admin_guide/manage_users.adoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Adding Users
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: Remember to add manual, unique IDs for headings (for example, check to make sure there's not some other heading in the repo named "adding-users").

File: admin_guide/manage_users.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+Once new users log in to OpenShift, an account is created for that user and the
+cluster administrator can
+link:../admin_guide/manage_authorization_policy.html[manage the access level of
+each user].
+
+== Deleting Users
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: It's nicer if we can avoid headings that just lead into a subheading. For example, if there's some intro or explanation we can add here before going into Improper Deletion.

File: admin_guide/manage_users.adoc [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+=== Improper Deletion
+
+OpenShift user configuration is stored in several locations within OpenShift.
+Regardless of the identity provider, OpenShift internally stores details like
+role-based access control (RBAC) information and group membership. To completely
+remove user information, this data must be removed in addition to the user
+account.
+
+If an account is improperly deleted, the following type of error can be seen,
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: Seen by who and when? I assume it's seen by the user/developer when attempting to authenticate or do anything?

File: admin_guide/manage_users.adoc [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+even though the identification provider already shows the user *myuser* as
+existing:
+
+====
+----
+{
+    "MESSAGE": "E0412 06:07:48.142862     465 login.go:141] Unable to authenticate password: Error creating or updating mapping for: &api.DefaultUserIdentityInfo{ProviderName:\"htpasswd_auth\", ProviderUserName:\"myuser\", Extra:map[string]string{}} due to users \"myuser\" not found",
+    "PRIORITY": "6",
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: We can probably trim this error message to just the "MESSAGE" here? I.e., just line 36.

File: admin_guide/manage_users.adoc [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+    "_UID": "0",
+}
+----
+====
+
+This error is generated because OpenShift sees the user data in the internal
+database that does not correspond to the external identity.
+
+In OpenShift, two object types contain user data outside the identification
+provider: type user and type identity.
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: We don't really refer to objects as "type <name>" anywhere else, so this feels weird here. What about just:

```
In OpenShift, two object types contain user data outside the identification provider: `*user*` and `*identity*`.
```

File: admin_guide/manage_users.adoc [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+Type user:
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: Maybe make these say what they're showing, e.g., "To get the current list of users:" (Same for "type identity" below)

File: admin_guide/manage_users.adoc [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+To delete a user, all three sources of the user must be removed:
+
+. Identification manager
+. User record
+. Identity record
+
+Once you complete these steps, a new account will be created in OpenShift
+when the user logs in again.
+
+==== Deleting the Identification Manager Entry
```
Comment (adellape) [inline admin_guide/manage_users.adoc]: Can we make these a procedure w/ numbered steps instead of individual subheadings per step?

**Comment by ahardin-rh** [comment]
Follow-up PR: https://github.com/openshift/openshift-docs/pull/1921

**Pull request merged**